tests/unit/lms/services/canvas_api/_authenticated_test.py

```diff
@@ -24,6 +24,21 @@ def test_send(self, authenticated_client, basic_client, oauth_token):
 
         assert result == basic_client.send.return_value
 
+    def test_send_raises_CanvasAPIAccessTokenError_if_we_dont_have_an_access_token_for_the_user(
+        self, authenticated_client, oauth2_token_service
+    ):
+        oauth2_token_service.get.side_effect = NoOAuth2Token()
+
+        with pytest.raises(
+            CanvasAPIAccessTokenError,
+            match="^We don't have a Canvas API access token for this user$",
```

Comment:
There are multiple ways that the method under test can raise `CanvasAPIAccessTokenError`, so I'm making sure that it raises the one that the test expects it to raise. Otherwise the code or test could be wrong and it could still pass